### ISSUE TITLE
Update mistralrs on PyPi to 0.1.26

### ISSUE THREAD

**Issue by pcpeasant** [EricLBuehler/mistral.rs#564]
**Describe the bug**
mistralrs 0.1.24 on PyPi fails to install with the following error:

```
Backend subprocess exited when trying to invoke build_wheel
  
  Running `maturin pep517 build-wheel -i /tmp/tmp0pr09x9p/.venv/bin/python --compatibility off`
      Updating crates.io index
      Updating git repository `https://github.com/EricLBuehler/candle.git`
  error: failed to get `candle-core` as a dependency of package `mistralrs-core v0.1.24 (/tmp/tmps25d5_iz/mistralrs-0.1.24/mistralrs-core)`
  
  Caused by:
    failed to load source for dependency `candle-core`
  
  Caused by:
    Unable to update https://github.com/EricLBuehler/candle.git?rev=b7a3e344#b7a3e344
  
  Caused by:
    object not found - no match for id (b7a3e34400000000000000000000000000000000); class=Odb (9); code=NotFound (-3)
  💥 maturin failed
    Caused by: Cargo metadata failed. Does your crate compile with `cargo build`?
    Caused by: `cargo metadata` exited with an error: 
  Error: command ['maturin', 'pep517', 'build-wheel', '-i', '/tmp/tmp0pr09x9p/.venv/bin/python', '--compatibility', 'off'] returned non-zero exit status 1
```

Meanwhile the 0.1.26 PyO3 bindings build without errors on my machine locally. Are the updates being held back from PyPi for a reason?

**Comment by EricLBuehler** [comment]
Thanks for letting me know @pcpeasant, #566 should fix this for our upcoming release 0.1.27.

**Comment by EricLBuehler** [comment]
@pcpeasant closing as I released v0.2.0 (https://github.com/EricLBuehler/mistral.rs/releases/tag/v0.2.0).
